doc/landing-page/index.html

```diff
@@ -0,0 +1,259 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Prism - Bootstrap 4 Open Source Landing Page Template</title>
+    <meta name="description" content="Prism is a beautiful Bootstrap 4 template for open-source landing pages."/>
+
+    <!--Google font-->
+    <link href="https://fonts.googleapis.com/css?family=K2D:300,400,500,700,800" rel="stylesheet">
+
+    <!-- Bootstrap CSS / Color Scheme -->
+    <link rel="stylesheet" href="css/bootstrap.css">
+</head>
+<body>
+
+<!--Header Section-->
+<section class="bg-gradient pt-5 pb-6">
+    <div class="container">
+        <div class="row">
+            <div class="col-12 d-flex flex-row align-items-center justify-content-between">
+                <div class="heading-brand">Algo Builder</div>
+                <a href="https://github.com/scale-it/algo-builder" class="btn btn-success svg-icon">
+                    <em class="mr-2" data-feather="github"></em>
+                    Github
+                </a>
+            </div>
+        </div>
+        <div class="row mt-6" style="margin-bottom: 50px;">
+            <div class="col-md-10 mx-auto text-center">
+                <h1>Framework to automate development of Algorand Assets and Smart Contracts</h1>
+            <p class="col-md-12 mx-auto mb-2 text-center" style="font-size: x-large;">Algo Builder is an trustworthy framework for Algorand dapps (Decentralized Applications).
+                    Its main goal is to make shipping Algorand applications simple, efficient, and scalable. Available packages:-
+                <div style="font-size:x-large; margin-left: 40px;">
```

Comment:
the links should have underline - this is a standard for displaying links.

Reply:
adding underline to all links (`<a>...</a>`)